### DISCUSSION TITLE
Filter out `_index.{{locale}}.md` from collection

### DISCUSSION THREAD

**Discussion by jbelien** [sveltia/sveltia-cms#259] [Q&A] [answered]
I have my "posts" in the same folder as my `_index.md` (using locale).

I've found in https://github.com/sveltia/sveltia-cms/issues/206#issuecomment-2379549051 that `_index.md` should be omitted by default and that's exactly what I what to achieve but it doesn't seem to apply to localized `_index.md` with `_index.{{locale}}.md` structure.

Is there an easy way to filter out ?

Thanks!

---

![image](https://github.com/user-attachments/assets/e0c09fa1-e583-4f42-952e-129c72d2c3af)

Files structure:
```text
content/news/
├── 20240825-paragon-downtime.en.md
├── 20240825-paragon-downtime.fr.md
├── 20240825-paragon-downtime.nl.md
├── 20241016-this-is-a-test.en.md
├── 20241121-no-translations-in-pgn-support-admin.de.md
├── 20241121-no-translations-in-pgn-support-admin.en.md
├── 20241121-no-translations-in-pgn-support-admin.fr.md
├── 20241121-no-translations-in-pgn-support-admin.nl.md
├── _index.en.md
├── _index.fr.md
└── _index.nl.md
```

**Comment by kyoshino** [comment]
I’ll take care of it 🙂

**Comment by kyoshino** [comment, marked as answer]
Fixed in [v0.47.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.47.0).

**Comment by jbelien** [comment]
That's what I call world class support!
THANK YOU (once again) @kyoshino 👍
